tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
+    renderSquare(i) {
+        return (
+            <Square
+                value={this.props.squares[i]}
+                onClick={() => {this.props.onClick(i)}}
```

Comment:
create event handler for it

Reply:
I made a functional component.
```suggestion
function Board({squares, onClick}) {
  const squareComponents = squares.map((square, index) => {
    return (
      <Square
        value={squares[index]}
        onClick={() => {onClick(index)}}
        key={index}
      />
    )
  }); 
```